src/tests/Microsoft.PowerFx.Core.Tests/DisplayNameTests.cs

```diff
@@ -107,5 +107,48 @@ public void ConvertToInvariantNamesNoNames()
 
             Assert.Equal("If(B, Num, 1234)", displayExpressions);
         }
+
+        [Theory]
```

Comment:
>Theory [](http://example.com/codeflow?start=9&length=6)

A few more tests: 
Add a  case that shows we're not just doing string.replace
"If(B, Text(B), "B")",  B-->"A",  ==> "If(A, Text(A), "B")" 

"B", B-->'B b'  ==> "'B b'"  // add spaces in identifier